### ISSUE TITLE
Create app "Festival"

### ISSUE THREAD

**Issue by GrandSchtroumpf** [blockframes/blockframes#2215]
# Application
The "festival" application is a marketplace "all rights" where members of organizations can find each other. It will be based on the catalog app. Here are the modifications to apply: 

### Header : 
- [x] Replace Archipel Content by new logo (Archipel Content, Film Market Edition) in the header
Delete « Cart » logo in the header

### Menu : 
- [x] Delete « My Selection » and « Offers and Deals »

### Home page : 
- [x] Change banner text (« Archipel Content » > delete. / « The only marketplace offering package deals » > « Your online film market ». / « For TV, VOD and ancillary rights » > « brought to you by Archipel Content »)
- [x] Change banner CTA : « View the Library » > « Search for films »
- [x] Sort by production status. Possible to add carousels ? If so, 4 status : Pre-Production / In Production / Post Production / Completed), if not, keep the 3 : Pre Production & In Production / Post Production / Completed)

### Library page :
**Filters** :
- [x] Delete « Freshness » filter
- [x] Delete the title « Title metadata » if possible
- [x] Delete the whole Avails part
- [x] Add a « Seller » filter
- [x] (Correct typo under « Filters » (if not already done))

**Films cards** :
- [ ] If not too complicated, add seller’s name on film card (in capital letters). Clickable if possible (to lead us to the org page of the seller.

### Film page :
- [x] Delete Avails tab
- [x] Delete « Switch to avails » CTA
- [x] Add Seller’s logo (clickable > would lead us to org page). Maybe on the top right, under or over « Add to wishlist »
- [x] Promotional elements : possible to have an integrated player that pops up when we click ? I don’t know if that’s the same as embeding a video or more complicated. If too complicated, forget about it.
- [x] Add « Screening information » in the promotional elements card (Free text : For the seller to specify if their will be a screening and when, with a link to « screening room ») 

### TODO Organization page (inspired by Film page design)
**Header banner** :
- [x] Instead of film poster, put org logo
- [x] Replace first line (MOVIE | 96 MIN | COMEDY, HORROR, THRILLER | ENGLISH) by company activity
- [x] Replace film title by company name
- [x] Replace original title line by company country 
- [x] Replace directors line by the text « Company description » (as a title for next field)
- [x] Replace countries of origin line by Company description (cf comments below for the global app’s ORGANIZATION PAGE)
- [x] Replace « Switch to Avails » CTA by « Get in touch » (> opens an email template if possible with the company contact as recipient)
- [x] Delete « Add to wishlist » CTA. 

**1rst Tab** :
- [x] Rename tab (Main information) to « Line-up »
- [x] Replace all content below by seller’s film cards

**2nd Tab** :
- [x] Rename tab (Avails) to « Contacts »
- [x] Left card (instead of avails filter) : should show company address + company website
- [x] Right card (instead of the map and stuff) : should show a list of contacts (organization members : First name, Surname, email adress)

### TODO: « Screening Room » page. 
No need for lot of design : Black background, title, with a video player in the center (embeded with Vimeo Livestreaming player). 
Eventually a link to the « screening schedule » page under the player, if we do integrate a calendar.

### _IF WE INTEGRATE A CALENDAR_ :  TODO « Screening Schedule » page. 
No need for lot of design. 1 Title, 1 Calendar. 

### _IF WE INTEGRATE A CALENDAR_ : TODO « My Schedules » page : 
No need for lot of design. Should show 2 calendars with 1 title each : 1. « My Meetings » / 2. « My Screenings »

### Menu : 
- [x] Delete « Offers and Deals »
- [x] _IF WE INTEGRATE A CALENDAR_ : Add « My Schedules »

#### Instruction page 
- [x] Delete Step 3 (Legal Information). If too complicated, replace by « Summary »
- [x] Change Intructions text : to « You can fill it out in one go or over time. You may navigate between steps, but all the essential elements (marked with an asterisk) must be completed before uploading your title on the marketplace. »
- [x] Change submission process/behaviour : when seller submits title, it should go online directly and be available on the marketplace (without needing to be validated by an Archipel team member) 

#### Main information
**Title information** :
- [ ] Delete « Freshness »
- [x] Delete « Release media »
- [ ] If possible, in « Release date » : add free text field (in case they don’t have a precise date)
- [ ] Running time : add option « TBC » (instead of just numbers)
**Budget Quotas & Critics** : 
- [x] Delete « Box office »

#### Media
**Promotional Images**
- [ ] Delete the whole « Banner image » card
- [ ] Files & links :
- [ ] File links : Rename to « Downloadable links »

#### Legal information : 
- [x] DELETE page

#### Summary
**Summary and Sumbit** :
- [x] Correct typo if not corrected yet : Rename to « Summary & Submission »
- [ ] Delete previously deleted elements

## ORGANIZATION PAGE (account admin) :
- [ ] Delete the whole « Bank information » card
- [ ] In « Legal Entity » card : 
- [x] Rename « Legal Entity » to « Company description »
- [x] In « Activity Description » replace open text field to dropdown list. If possible, it would be great if they could chose different options.
Here is the list : _Production, International Sales, Distribution, Television Broadcast, VOD Platform, Theatrical Distribution, Buyers’ Rep, Film Festival, Film Fund, Film Festival, Film Library_
- [ ] Replace « Fiscal Number » by « Company description » (should be an open field for text).

**Comment by Jackseed** [comment]
Close this issue, the movie & org form updates will be done later.
